iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/node/NodeManager.java

```diff
@@ -421,9 +421,11 @@ public TConfigNodeRegisterResp registerConfigNode(TConfigNodeRegisterReq req) {
     int nodeId = nodeInfo.generateNextNodeId();
     req.getConfigNodeLocation().setConfigNodeId(nodeId);
     configManager.getProcedureManager().addConfigNode(req);
+    String clusterId = configManager.getClusterManager().getClusterId();
     return new TConfigNodeRegisterResp()
         .setStatus(ClusterNodeStartUtils.ACCEPT_NODE_REGISTRATION)
-        .setConfigNodeId(nodeId);
+        .setConfigNodeId(nodeId)
+        .setClusterId(clusterId);
```

Comment:
Not quite necessary here, CN's consensus layer will automatically sync cluster id

Reply:
changed.